charts/hivemq-platform-operator/values.yaml

```diff
@@ -18,9 +18,26 @@ resources:
   # Set to specify the ephemeral-storage needed for managing local ephemeral storage in the Operator pod.
   # ephemeralStorage: 1Gi
 
-# Configures the log level, possible values are: "TRACE", "DEBUG", "INFO", "WARN", "ERROR"
+# Configures the log level for the HiveMQ Platform Operator.
+# Possible values are: "TRACE", "DEBUG", "INFO", "WARN", "ERROR"
 logLevel: INFO
 
+# Configures the Java Operator SDK log level for the HiveMQ Platform Operator.
+# Possible values are: "TRACE", "DEBUG", "INFO", "WARN", "ERROR"
+networkLogLevel: WARN
+
+# Configures the Java Operator SDK log level for the HiveMQ Platform Operator.
+# Possible values are: "TRACE", "DEBUG", "INFO", "WARN", "ERROR"
+sdkLogLevel: INFO
+
+# Configures whether to log the HiveMQ Platform Operator configuration at startup. Sensitive values are redacted.
+logConfiguration: false
```

Comment:
I would not expose these 3 options and just keep them in the `deployment.yml` template as they are, keeping the `.Values.xxx | default "..."` so they can be easily overridden in case we need to debug something.

Those are not needed for production and might just be confusing.